browser/tor/onion_location_tab_helper.h

```diff
@@ -0,0 +1,41 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_TOR_ONION_LOCATION_TAB_HELPER_H_
+#define BRAVE_BROWSER_TOR_ONION_LOCATION_TAB_HELPER_H_
+
+#include "content/public/browser/web_contents_observer.h"
+#include "content/public/browser/web_contents_user_data.h"
+
+namespace tor {
+
+class OnionLocationTabHelper
+    : public content::WebContentsObserver,
```

Comment:
Does this need to be a `WebContentsObserver` if it doesn't actually observe anything? Unless I'm missing something, it seems like it only needs to be a `WebContentsUserData<OnionLocationTabHelper>`

Reply:
yep, it doesn't observe anything. I will remove it.

Reply:
removed in 68c9215c99860a8b67d72a14c1f144f4d3e04963